### PR TITLE
Install jupyter spec 

### LINKED ISSUE #30: Installing r-irkernel is not sufficient to enable in jupyter lab
### Comment:

The conda installation does not perform step 2 of the installation:
https://irkernel.github.io/installation/

This leads to much confusion.

### LINKED ISSUE #31: r-irkernel cannot properly integrate into conda environment jupyter lab, uses incorrect path
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

I'm trying to finish the install for r-irkernel:
https://irkernel.github.io/installation/

Inside an environment:
```
$ which R
/opt/quarantine/software/R/4.1.3/r-base/bin/R
$ which jupyter
/opt/quarantine/software/R/4.1.3/r-base/bin/jupyter
$ conda list r-irkernel
```

But trying to install "for all users" in R, tries to modify local system files
```
> > IRkernel::installspec(user = FALSE)
[Errno 13] Permission denied: '/usr/local/share/jupyter'
```

### Installed packages

```shell
Github complains the comment is too long
```

### Environment info

```shell
active environment : /opt/quarantine/software/R/4.1.3/r-base
    active env location : /opt/quarantine/software/R/4.1.3/r-base
            shell level : 2
       user config file : /home/cic/devgab/.condarc
 populated config files : /opt/quarantine/software/R/4.1.3/r-base/.condarc
          conda version : 4.12.0
    conda-build version : 3.21.8
         python version : 3.9.12.final.0
       virtual packages : __cuda=9.1=0
                          __linux=5.15.0=0
                          __glibc=2.35=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /opt/quarantine/software/anaconda/2022.05/install  (writable)
      conda av data dir : /opt/quarantine/software/anaconda/2022.05/install/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://conda.anaconda.org/bioconda/linux-64
                          https://conda.anaconda.org/bioconda/noarch
          package cache : /opt/quarantine/software/anaconda/2022.05/install/pkgs
                          /home/cic/devgab/.conda/pkgs
       envs directories : /opt/quarantine/software/anaconda/2022.05/install/envs
                          /home/cic/devgab/.conda/envs
               platform : linux-64
             user-agent : conda/4.12.0 requests/2.27.1 CPython/3.9.12 Linux/5.15.0-30-generic ubuntu/22.04 glibc/2.35
                UID:GID : 1111:1001
             netrc file : None
           offline mode : False
```

### PR DESCRIPTION
**Pull request by philip-khor**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by philip-khor** [comment]
@conda-forge-admin, please rerender

**Comment by philip-khor** [comment]
@conda-forge-admin, please rerender

**Comment by philip-khor** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,6 +33,7 @@ requirements:
     - r-pbdzmq >=0.2_1
     - r-repr >=0.4.99
     - r-uuid
+    - jupyter
```
Comment (mfansler) [inline recipe/meta.yaml:36]: Including a `jupyter` dependency in the kernel undermines the modularity of kernels. The point is that kernels can be loaded into Jupyter without having Jupyter coinstalled in the same environment.

**Review by mfansler — COMMENTED** [review]
This should not have been merge without a review, as it introduces changes to how the package behaves.

File: recipe/build.sh [hunk 2]
```diff
@@ -1,40 +1,5 @@
...
-            install_name_tool -change /usr/lib/libcurl.4.dylib "$PREFIX"/lib/libcurl.4.dylib $SHARED_LIB || true
-            install_name_tool -change /usr/lib/libc++.1.dylib "$PREFIX"/lib/libc++.1.dylib $SHARED_LIB || true
-            install_name_tool -change /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libc++.1.dylib "$PREFIX"/lib/libc++.1.dylib $SHARED_LIB || true
-          done
-        popd
-      done
-    popd
-  fi
-fi
+$R -e "IRkernel::installspec(sys_prefix = TRUE)"
```
Comment (mfansler) [inline recipe/build.sh:5]: Even if this should be done (I don't think it should), it is not a build step, but rather something to be done in a `post-link` script and cleaned up in a `pre-unlink` script.

**Comment by philip-khor** [comment]
Apoloigies for the poor judgement. I've PRed to reverse the change pending more opportunity for review.

**Pull request merged**